sanity/keylime-secure_mount/test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+SECURE_DIR="/var/lib/keylime/secure"
+KEYLIME_UNIT_FILE="/etc/systemd/system/keylime_agent.service"
+if [ -n "$KEYLIME_UNIT_FILE" ]; then 
+    KEYLIME_UNIT_FILE="/usr/lib/systemd/system/keylime_agent.service"
+fi
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+
+        # update /etc/keylime.conf
+        limeBackupConfig
+        rlFileBackup $KEYLIME_UNIT_FILE
+        rlRun "limeUpdateConf logger_root level DEBUG"
+        rlRun "limeUpdateConf logger_keylime level DEBUG"
+        rlRun "limeUpdateConf handler_consoleHandler level DEBUG"
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            rlRun "limeCondStartAbrmd"
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        sleep 5
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Verify that agent creates mount point when not present and mounts tmpfs."
+        rlRun "limeStartAgent"
+        rlRun "limeStopAgent"
+        rlRun "umount $SECURE_DIR"
+        rlRun "rm -rf $SECURE_DIR"
+        sed -i '/var-lib-keylime-secure.mount/d' $KEYLIME_UNIT_FILE
```

Comment:
Just thinking, if you are doing these unit file updates just to avoid mount point creation by systemd and you want to test agent itself is able to do the mount, why aren't you starting `keylime_agent` directly (not through systemd)? It should help you do avoid doing all these systemd related changes.